containers/layout/Layout.tsx

```diff
@@ -12,29 +12,27 @@ import { useAuth } from 'hooks/useAuth';
 import { useRouter } from 'next/router';
 import { FirebaseError } from 'lib/types/auth';
 import Error from 'components/error';
+import { withTranslation } from 'i18n.config';
+import { layoutVar } from 'lib/context-variables';
```

Comment:
in the Apollo docs, they keep reactive vars in a cache file instead. Makes sense, because the cache is where they live. 
We could do the same:
add cache.ts to our apollo folder
in cache.ts:
- define and export the reactive var
- const cache = new InMemoryCache()
- export default cache

In apollo/client.ts instead of instantiating new InMemoryCache, import cache from cache.ts
Imports of the reactiVar would then be from apollo/chache.ts

Reply:
ah nice, ok. I will try that out 👍

Reply:
I did that now